src/localization/src/translations/ru.ts

```diff
@@ -292,7 +294,7 @@ export default {
       shareUriTitle: 'Поделиться URL карты',
       shareUriSubtitle: 'Создать URL карты, чтобы поделиться с другими',
       cloudTitle: 'Облачное хранилище',
-      cloudSubtitle: 'Войдите и загрузите данные карты в свое личное облачное хранилище',
+      cloudSubtitle: 'Войдите и загрузите данные карты в свое лич��ое облачное хранилище',
```

Comment:
looks like some of the cyrillic characters were corrupted here

Reply:
Yes, that's something wrong when saving the file. Will fix it :)